src/main/java/com/dutact/web/auth/factors/admin/AdminAuthController.java

```diff
@@ -0,0 +1,50 @@
+package com.dutact.web.auth.factors.admin;
+
+import com.dutact.web.auth.dto.LoginDto;
+import com.dutact.web.auth.dto.ResponseToken;
+import com.dutact.web.auth.exception.InvalidLoginCredentialsException;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.ExampleObject;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import jakarta.validation.Valid;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Map;
+
+@RestController
+public class AdminAuthController {
+    private final AdminAuthService adminAuthService;
+
+    public AdminAuthController(AdminAuthService adminAuthService) {
+        this.adminAuthService = adminAuthService;
+    }
+    @Operation(summary = "Login for admin users", description = "Authenticate an admin user and return a JWT token")
```

Comment:
Why write summary and description, the endpoint `/api/admin/login` is self explanatory

Reply:
Ok